source/development/wazuh-cluster.rst

```diff
@@ -1,8 +1,16 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
+.. Section marks used on this document:
```

Comment:
It's not the current style of the documentation to leave notes to remember, and this document is not a general reference but a particular topic. In addition, this is not valid since it starts with h1. It is suggested to remove every one of these comments.